### ISSUE TITLE
Reproducing results on SIIM-ISIC 2019 Dataset

### ISSUE THREAD

**Issue by kushiluv** [pranavsinghps1/CASS#1]
Hey, i'm trying to reproduce the results of your paper but am unable to do so using the default cfg you provided in the CASS.ipynb.
Could you please tell me how can i finetune it.

**Comment by pranavsinghps1** [comment]
Hello, the end-to-end fine-tuning code is included in CASS.ipynb. Since the ISIC-2019 dataset didn't release the official test set, we are splitting the training set into three (train/val and test); your performance may vary depending on how you split the dataset. For our experimentation, we used 
```
X_train, X_val_test, y_train, y_val_test = train_test_split(df['image_name'], df['diagnosis'], test_size=0.2, random_state=77)

X_val, X_test, y_val, y_test = train_test_split(X_val_test, y_val_test, test_size=0.5, random_state=77)
```

Where `df` is the data frame consisting of images and corresponding labels.

**Comment by pranavsinghps1** [comment]
I am assuming the issue has been resolved, feel free to reopen it if need be. Thank you!
